### DISCUSSION TITLE
Training Log 2022-11-26

### DISCUSSION THREAD

**Discussion by zh-zheng** [OpenBMB/CPM-Live#250] [Training Logs 训练日志]
# CPM-Live Training Log (November, 26)

Time: November, 26 2022 19:00

Recorder: @zh-zheng

## Loss
- Begin: 2.14
- End: 2.13
	
## Completed Data
- $\approx$ 1856.73GB

## Average Grad Norm
- 2.18

## Progress
- 44.82%

## Comment

Since we reduced the weights of the model and the learning rate, the training loss has been decreasing. Very interesting!
